src/migration-scripts/container/2-to-3

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 VyOS maintainers and contributors <maintainers@vyos.io>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+# T7473: container: allow log-driver to be set per container
+
+from vyos.configtree import ConfigTree
+
+base = ['container', 'log-driver']
+
+def migrate(config: ConfigTree) -> None:
+
+    if not config.exists(base):
+        # Nothing to do
+        return
+    else:
+        config.delete(base)
```

Comment:
I don't think it's enough to delete the option. Suppose it was set to a non-default value. After the migration script, it will revert to the default, to the user's surprise.

I think the correct migration logic in this case is to set that option for every configured container if it was set to a non-default value at the global level in the old config.

Please correct me if I'm missing anything.

Reply:
My initial thought was to use a migration script to prevent errors. your suggestion is definitely more reasonable. I've fixed it according to your recommendation.